docs/proposal/20210823-gpu-support.md

```diff
@@ -0,0 +1,177 @@
+# GPU support in CAPV
```

Comment:
We had discussed updating the title of the proposal to explicitly call out the PCI Passthrough support, could you include that here?

Reply:
Had updated an outdated version in my previous commit. Updated the new one.